controllers/api/tweetController.js

```diff
@@ -70,7 +74,7 @@ const tweetController = {
 
```

Comment:
Including User of Reply
Maybe add a isMyReply: true or false for future editing Reply feature